lib/Sema/TypeCheckProtocolInference.cpp

```diff
@@ -1966,8 +1966,9 @@ auto AssociatedTypeInference::solve(ConformanceChecker &checker)
       if (replacement->hasDependentMember())
         return None;
 
-      if (replacement->hasArchetype())
+      if (replacement->hasArchetype()) {
```

Comment:
Useless edit here

Reply:
I thought LLVM style was always to brace if blocks. It's more maintainable regardless.